static/js/browser2/out_arc_section.tsx

```diff
@@ -141,17 +147,19 @@ export class OutArcSection extends React.Component<
             if (!(predicate in outArcsByPredicateAndProvenance)) {
               outArcsByPredicateAndProvenance[predicate] = {};
             }
+            const outArcsOfPredicate =
```

Comment:
Both "Predicate" and "Property" are used which is a bit in-consistent, might be using "property" throughout? And I feel the names here a bit too long, something like "outArcsByPropProv" might work as well.

Reply:
I think using either "predicate" or "property" will present the same problem. The data we get back from the triples api uses "predicate", but the data we get back from the propVals api uses "property". I believe this component mainly uses "predicate" right now, do you want me to change it to "property"?

Reply:
I see. That's a good point. We should use "property" for our next version of API so the naming are aligned. Then no need to do the conversion here.

Reply:
ok, sounds good! But, I'll shorten the names